omigami/feature_selector.py

```diff
@@ -304,10 +326,33 @@ def _evaluate_min_mid_and_max_features(
     def _select_best_features(
         self, repetition_results: FeatureSelectionResults
     ) -> SelectedFeatures:
-        self.results = self.post_processor.fetch_results(repetition_results)
+        self.results = self._fetch_results(repetition_results)
```

Comment:
why moving it here? (I'm not against it, just curious)

Reply:
it felt that it was better placed in the selector. there is no processing being done and since I needed to add the progressbar on it, it would look very ugly if this method was on the post processor